src/bootstrap/doc.rs

```diff
@@ -74,6 +74,7 @@ macro_rules! book {
 // and checking against it?).
 book!(
     CargoBook, "src/tools/cargo/src/doc", "cargo", submodule = "src/tools/cargo";
+    ClippyBook, "src/tools/clippy/book", "clippy", submodule = "src/tools/clippy";
```

Comment:
not a submodule, you can remove that bit.
```suggestion
    ClippyBook, "src/tools/clippy/book", "clippy";
```